docs/UserGuide/Appendix/SQL-Reference.md

```diff
@@ -716,9 +716,27 @@ In this situation, it will throws an exception if * corresponds to multiple sens
 
 ```
 
+* Regexp Statement
+
+Regexp Statement only supports regular expressions with data type of text and Java standard library style when matching
+```
+SELECT <SelectClause> FROM <FromClause> WHERE  <WhereClause>
+Select Clause : <Path> [COMMA <Path>]*
+FromClause : < PrefixPath > [COMMA < PrefixPath >]*
+WhereClause : andExpression (OPERATOR_OR andExpression)*
+andExpression : predicate (OPERATOR_AND predicate)*
+predicate : (suffixPath | fullPath) LIKE stringLiteral
+stringLiteral : SINGLE_QUOTE_STRING_LITERAL | DOUBLE_QUOTE_STRING_LITERAL
```

Comment:
here, `stringLiberal` should be regular expressions, rather than simple `SINGLE_QUOTE_STRING_LITERAL`.

Besides, a reference of Java regular expressions should be given.